.stickler.yml

```diff
@@ -8,6 +8,7 @@ linters:
 files:
   ignore:
       - 'dist/*'
+      - 'test/*'
```

Comment:
You should not hide test directory but ensure it meets eslint requirements.
You should not make changes from original config files as can be seen from this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from pathwright.

Please address.

Reply:
You should try addressing the Eslint issues instead!